.secrets.baseline

```diff
@@ -27443,10 +27456,9 @@
         "verified_result": null
       }
     ],
-    "dev/com.ibm.ws.openapi.ui/swagger-ui/dist/liberty-swagger-ui-bundle-bdf9b527ec6da92b71c5.js": [
+    "dev/com.ibm.ws.openapi.ui/swagger-ui/dist/liberty-swagger-ui-bundle-cd2c6b19d75012eb0aa3.js": [
       {
         "hashed_secret": "6a40fbb3bd3c0673c9bff6ad305c8181d648a9bb",
-        "is_secret": false,
```

Comment:
Nope..not sure how it was removed. I can add it back.

Reply:
I see the file name has been changed in the end (from **bdf9b527ec6da92b71c5.js** to **cd2c6b19d75012eb0aa3.js**).

Reply:
It's possible this file name changes periodically (when built or when pulled from upstream).  I suggest checking if @Azquelt to see if that's the case. If it is, and depending on how often this gets updated, it might make sense to add it to the ignore list.

Reply:
Yes, this is a minified javascript file which includes a hash of the file in the filename to ensure that browsers can cache the file, but won't wrongly use the cache if the file has changed.

Reply:
Ignoring `dev/com.ibm.ws.openapi.ui/swagger-ui/dist/liberty-swagger-ui-bundle-*.js` should be safe.